vue-app/src/components/button-footer.vue

```diff
@@ -43,6 +44,7 @@ export default {
     @click="handleClick"
     :data-index="dataIndex"
     :data-id="dataId"
+    type="button"
```

Comment:
added due to lint rule on accessibility, i.e must have a type, keep or remove? @johnpapa